src/RestrictedPython/transformer.py

```diff
@@ -270,7 +275,8 @@ def gen_unpack_wrapper(self, node, target, ctx='store'):
         # 'wrapper' takes care to protect sequence unpacking with _getiter_.
         converter = self.protect_unpack_sequence(
             target,
-            ast.Name(tmp_name, ast.Load()))
+            ast.Name(tmp_name, ast.Load()),
```

Comment:
A trailing comma in lists, tuples or dicts is okay but having it in each function call is nonsense.